diff --git a/node_modules/expo-router/_ctx.ios.js b/node_modules/expo-router/_ctx.ios.js
index b30ee6c..178bbfb 100644
--- a/node_modules/expo-router/_ctx.ios.js
+++ b/node_modules/expo-router/_ctx.ios.js
@@ -1,6 +1,6 @@
 export const ctx = require.context(
-  process.env.EXPO_ROUTER_APP_ROOT,
+  "../../app",
   true,
   /^(?:\.\/)(?!(?:(?:(?:.*\+api)|(?:\+html)|(?:\+middleware)))\.[tj]sx?$).*(?:\.android|\.web)?\.[tj]sx?$/,
-  process.env.EXPO_ROUTER_IMPORT_MODE
+  "sync"
 );
diff --git a/node_modules/expo-router/build/views/Sitemap.js b/node_modules/expo-router/build/views/Sitemap.js
index 4680d37..a34faa1 100644
--- a/node_modules/expo-router/build/views/Sitemap.js
+++ b/node_modules/expo-router/build/views/Sitemap.js
@@ -116,16 +116,16 @@ function SitemapItemPressable({ style, leftIcon, rightIcon, filename, level, inf
     </Pressable_1.Pressable>);
 }
 function FileIcon() {
-    return <react_native_1.Image style={styles.image} source={require('expo-router/assets/file.png')}/>;
+    return <react_native_1.Image style={styles.image} source={require('../../../../assets/file.png')}/>;
 }
 function PkgIcon() {
-    return <react_native_1.Image style={styles.image} source={require('expo-router/assets/pkg.png')}/>;
+    return <react_native_1.Image style={styles.image} source={require('../../../../assets/pkg.png')}/>;
 }
 function ForwardIcon() {
-    return <react_native_1.Image style={styles.image} source={require('expo-router/assets/forward.png')}/>;
+    return <react_native_1.Image style={styles.image} source={require('../../../../assets/forward.png')}/>;
 }
 function SitemapIcon() {
-    return <react_native_1.Image style={styles.image} source={require('expo-router/assets/sitemap.png')}/>;
+    return <react_native_1.Image style={styles.image} source={require('../../../../assets/sitemap.png')}/>;
 }
 function ArrowIcon({ rotation = 0 }) {
     return (<react_native_1.Image style={[
@@ -133,7 +133,7 @@ function ArrowIcon({ rotation = 0 }) {
             {
                 transform: [{ rotate: `${rotation}deg` }],
             },
-        ]} source={require('expo-router/assets/arrow_down.png')}/>);
+        ]} source={require('../../../../assets/arrow_down.png')}/>);
 }
 function SystemInfo() {
     const getHermesVersion = () => {
